tests/template_tests/syntax_tests/test_i18n.py

```diff
@@ -342,17 +342,38 @@ def test_i18n31(self):
         )
 
     @setup({'i18n32': '{% load i18n %}{{ "hu"|language_name }} '
-                      '{{ "hu"|language_name_local }} {{ "hu"|language_bidi }}'})
+                      '{{ "hu"|language_name_local }} {{ "hu"|language_bidi }} '
+                      '{{ "hu"|language_name_translated }}'})
     def test_i18n32(self):
         output = self.engine.render_to_string('i18n32')
-        self.assertEqual(output, 'Hungarian Magyar False')
+        self.assertEqual(
```

Comment:
I'd opt for longer lines for most of these `assertEqual`s (we prefer longer lines up to 119 chars when it helps readability).